ds3-autogen-python/src/main/java/com/spectralogic/ds3autogen/python/generators/response/HeadObjectResponseGenerator.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014-2018 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3autogen.python.generators.response;
+
+import com.spectralogic.ds3autogen.api.models.apispec.Ds3Request;
+
+import static com.spectralogic.ds3autogen.python.helpers.PythonHelper.pythonIndent;
+
+public class HeadObjectResponseGenerator extends HeadResponseGenerator {
+
+    /**
+     * Gets the python code that will parse the response payload
+     */
+    @Override
+    public String toParseResponsePayload(final Ds3Request ds3Request) {
+        return "if self.response.status == 200:\n" +
```

Comment:
You can replace this with a kotlin string literal by using `'''` and you'll be able to have the correct indenting without having to use the `pythonIndent` method.